user_single.ml

```diff
@@ -239,175 +238,183 @@ let user_single_layout (user : User_model.user) unikernels policies current_time
             section
               ~a:[ a_id "settings"; a_class [ "my-5 tab-pane hidden" ] ]
               [
-                table
-                  ~a:
-                    [
-                      a_class
-                        [ "table-auto min-w-full divide-y divide-gray-200" ];
-                    ]
-                  ~thead:
-                    (thead
-                       [
-                         tr
+                (match policy with
+                | None -> h2 [ txt "No policy" ]
```

Comment:
I'm not sure about this line -- I think if we don't see any policy, we don't need to display any.

Reply:
I think it's fine. We can complement it with a button to set new policies